assets/sass/patterns/atoms/list.scss

```diff
@@ -6,13 +6,6 @@
   padding-left: 0.75rem;
 }
 
-// Had to put this here as list.scss overrides table.scss
```

Comment:
Does this no longer apply?

Reply:
Have you got the latest version as I removed this on Friday?